### ISSUE TITLE
Router documentation is missing import of implicits

### ISSUE THREAD

**Issue by jypma** [japgolly/scalajs-react#404]
The documentation page on the router (`ROUTER.md`) fails to mention that `render(MyComponent())` won't compile without the following import:
```
import japgolly.scalajs.react.vdom.html_<^._
```
which isn't necessary for the rest to compile if there is no direct usage of vdom. The docs could be updated to mention the extra import (which allows `scalajs.react.component.Js.UmountedWithRoot[...junk...]` to be coerced to a `VdomElement` into `render`)

**Comment by japgolly** [comment]
Indeed!

**Comment by japgolly** [comment]
Actually, you can import less:

```scala
import japgolly.scalajs.react.vdom.Implicits._
```

I made mention of this in the 1.0.0 changelog but I agree, it should be in the router doc too.

**Comment by japgolly** [comment]
Closed via 227f65b9f9d62e6e40085df2ef6a7da7d9dca50b
